airbytehq/airbyte
# Bump Airbyte version from 0.39.26-alpha to 0.39.27-alpha

*IMPORTANT: Only merge if the platform build is passing!*

Changelog:

dde7d1b70e Wait indefinitely if connection is not active (#14200)
11d4cc7adb Handle null reset source config (#14202)
8ebd5a0ab0 Improve documentation for Postgres Source (#13830)
79f7d060db 🐛 destination-redshift: use s3 bucket path for s3 cleanup (#14190)
7565524f43 🪟 :art: Show credit usage on chart's specific day (#13503)
9c289516ee Harshith/test pr 13118 (#14192)
7a15f2d4a5 🎉 Source Github: secondary rate limits has to retry (#13955)
5f4dacc1ca Source Tiktok Marketing: Videometrics (#13650)
76e50fdd7b 🐛 Source Zendesk Support: fixed "Retry-After" non integer value (#14112)
26a35af781 🎉 Destination connectors: Improved "SecondSync" checks in Standard Destination Acceptance tests (#14184)
a0f5655e83 Fix StatePersistence Legacy read/write (#14129)
f918d3295b Publish acceptance test utils maven artifact (#14142)
16d99b9b0c 🐛 Destination Redshift: use s3 bucket path for s3 staging operations (#13916)
a6bb6e0ead 🎉 Introduce Google Analytics Data API source (#12701)
1d5ff44061 Source Amazon Ads: increase timeout for SAT (#14167)
c113f246a8 Source Salesforce: fix customIntegrationTest for SAT (#14172)
062b12f1ba 🎉 Base Norrmalization: clean-up Redshift `tmp_schemas` after SAT (#14015)
4bf1ab17fb source-orbit: add definition and specs (#14189)
626e115e31 🎉 New Source: Orbit.love (#13390)
5c8969f52b Use correct bash comment symbol. (#14183)
93181daa7b Increase instance termination time limit to 3 hours to accommodate connector builds. (#14181)
070b80f448 remove VersionMismatchServer (#14076)
d19cbefae1 🎉 Source BingAds: expose hourly/daily/weekly/monthly options from configuration (#13801)
501a1c3e6a Source Twilio: adopt best practices (#14000)
9828246e12 :bug: Source Intercom: extend `Contacts` schema with new properties (#14099)
3ce1ee96df 🐛 Source Hubspot: fixed SAT test, commented out expected_records (#14140)
b5ea9acdd7 13523 Fix integration tests destination-cassandra Mac OS (#14134)
e7cce38dad 13541 Fixed integration tests source-db2 Mac OS (#14133)
a89130237f 🎉 octavia-cli: Add ability to get existing resources (#13254)

Instructions:
- *SQUASH MERGE* this PR - this is necessary to ensure the automated Create Release action is triggered.
- Double check that the Create Release action was triggered and ran successfully on the commit to master (this should only take a few seconds).
- If the Create Release action failed due to a transient issue, retry the action. If it failed due to a non-transient issue, you will need to manually create a release by following these steps:
 1. Pull most recent version of master
 2. Run ./tools/bin/tag_version.sh
 3. Create a GitHub release with the changelog
